### ISSUE TITLE
[Bug] Downloading and verifying add-on notification not available to Talkback users

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#9794]
## Summary
When you activate the button to install an add-on from the add-ons manager in Fenix, a sighted user gets a status notification bar at the bottom of the screen that reads "Downloading and verifying add-on..." which is replaced by a differently styled notification at the bottom of the screen that reads "Successfully installed uBlock Origin". The first notification is not read at all to Talkback. The second notification is.

## Steps to reproduce
Enable Talkback
Install the uBlock Origin add-on
Listen for the downloading and verifying add-on message. Try to swipe to it. Nothing. 

### Expected behavior
Talkback should announce "Downloading and verifying add-on..." 

### Actual behavior
Talkback users do not get a status notification. 

### Device information

* Android device: Galaxy S10e
* Fenix version: Sunday 4/5 @ 6:10 PM

**Comment by asadotzler** [comment]
This is an accessibility P2. The overall feature is available but is more difficult to use because of this bug.

**Comment by mcarare** [comment]
On a Huawei P10 with Android 9 I can swipe to first notification and Talkback can read it. Are you referring to the fact that it is not read automatically like the second one?

**Comment by asadotzler** [comment]
You are correct. I can swipe to the Downloading notification and it is read. It's not read automatically like the Successfully installed notification.

**Comment by sv-sdeiac** [comment]
Hi, I can confirm that this issue is fixed using the latest version of Firefox Preview Nightly 5/6 #21270609.
Now the download notification is read automatically when it appears.

Devices:
- Google Pixel 3 (Android 10);
- Samsung Galaxy Note 8 (Android 9).
